pkg/apis/integreatly/v1alpha1/installation_types.go

```diff
@@ -101,13 +113,14 @@ type PullSecretSpec struct {
 // InstallationStatus defines the observed state of Installation
 // +k8s:openapi-gen=true
 type InstallationStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// INSERT ADDITIONAL STATUS FIELDS - define observed state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 	Stages           map[StageName]*InstallationStageStatus `json:"stages"`
 	PreflightStatus  PreflightStatus                        `json:"preflightStatus,omitempty"`
 	PreflightMessage string                                 `json:"preflightMessage,omitempty"`
 	LastError        string                                 `json:"lastError"`
+	SMTPEnabled      bool                                   `json:"smtpEnabled,omitempty"`
```

Comment:
Should I change this to be required (remove the `,omitempty`)?

Reply:
Can the service be enabled at a later date?, if not then I'd say to have this as required, otherwise I don't think it makes that much of a difference

Reply:
This is set to true based on the Secret being found, and it will block for the _managed_ Installation type, but won't for the workshop Installation type.